x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/numeric_type.tsx

```diff
@@ -44,6 +44,45 @@ interface Props {
 export const NumericType = ({ field }: Props) => {
   return (
     <>
+      {/* scaling_factor only applies to scaled_float numeric type*/}
+      <FormDataProvider pathsToWatch="subType">
+        {formData =>
+          formData.subType === 'scaled_float' ? (
+            <EditFieldSection>
+              <EditFieldFormRow
+                title={
+                  <h3>
+                    {i18n.translate('xpack.idxMgmt.mappingsEditor.scalingFactorFieldTitle', {
+                      defaultMessage: 'Scaling factor',
+                    })}
+                  </h3>
+                }
+                description={i18n.translate(
+                  'xpack.idxMgmt.mappingsEditor.scalingFactorFieldDescription',
+                  {
+                    defaultMessage: 'The scaling factor to use when encoding values.',
+                  }
+                )}
+                withToggle={false}
+              >
+                <UseField
+                  path="scaling_factor"
+                  config={getFieldConfig('scaling_factor')}
+                  component={RangeField}
```

Comment:
I think this should just be a number input. The range has a min and max, but the docs don't indicate that there is a maximum value for this field. One of the examples has a scaling factor of 100.

Reply:
Good point. I agree a number input makes more sense here.